### ISSUE TITLE
[FEATURE] Test IPC large message IO whilst playback and capture are active

### ISSUE THREAD

**Issue by lgirdwood** [thesofproject/sof-test#847]
**Is your feature request related to a problem? Please describe.**
Intermittent issue when sending large config IPC data to component whilst playback and capture active. We are not testing large IPC messages in any test IIUC today. See https://github.com/thesofproject/sof/issues/4769

**Describe the solution you'd like**
Test script for large config IPC. Dummy WOV or any other module that takes large config module could be IPC target. Also an option to stress the large IPC to send 1000s of messages.

**Comment by lgirdwood** [comment]
@singalsu which common processing component accepts large IPC config data today ? This could be the test target as it wont require topology updates ?

**Comment by plbossart** [comment]
@lgirdwood We don't have such components in most topologies used in CI, the only one I see is CML_HEL_RT5682

So before we come-up with a test we need to have something to test, maybe by adding a EQ to the nocodec topologies after the mixer?

**Comment by lgirdwood** [comment]
I think IIR or FIR is one such component and IIUC we use it on some HDA pipes for pop removal ?

**Comment by plbossart** [comment]
that's correct, we use EQIIR in all nocodec, HDAudio and SoundWire topologies on the input paths.

**Comment by singalsu** [comment]
> @singalsu which common processing component accepts large IPC config data today ? This could be the test target as it wont require topology updates ?

FIR component accepts large config, e.g. https://github.com/thesofproject/sof/blob/main/tools/ctl/eq_fir_loudness.txt or https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/m4/eq_fir_coef_loudness.m4 .

**Comment by plbossart** [comment]
@singalsu does the component accept changes to the config *WHILE* audio is streaming? I thought we had some restrictions.

**Comment by marc-hb** [comment]
Note we already have a check-ipc-flood.sh test but I don't know what it does. It's skipped on Zephyr with "need a debug firmware"
